versions.lock

```diff
@@ -77,7 +77,7 @@ com.squareup.okio:okio:1.17.2 (1 constraints: 850cc309)
 commons-logging:commons-logging:1.2 (2 constraints: 8215ead1)
 de.erichseifert.vectorgraphics2d:VectorGraphics2D:0.13 (1 constraints: 8c0a80bb)
 de.rototor.pdfbox:graphics2d:0.25 (1 constraints: 8f0a84bb)
-io.undertow:undertow-core:2.2.18.Final (1 constraints: 57074361)
+io.undertow:undertow-core:2.2.24.Final (1 constraints: 54072f61)
```

Comment:
do not merge yet, will hold until com.palantir.conjure.java & com.palantir.conjure.java.runtime bumps